cmd/kuiperd/main.go

```diff
@@ -15,14 +15,45 @@
 package main
 
 import (
+	"flag"
+
+	"github.com/lf-edge/ekuiper/internal/conf"
 	"github.com/lf-edge/ekuiper/internal/server"
 )
 
+var Version = "unknown"
```

Comment:
LoadFileType must not be deleted. It is actually passed in compile time. Check MakeFile and search LoadFileType. If we remove this, it will be a breaking change.